src/documents/templates/registration/login.html

```diff
@@ -12,7 +12,7 @@
     <meta name="generator" content="Jekyll v4.1.1">
     <meta name="robots" content="noindex,nofollow">
 
-    <title>{% translate "Paperless-ng sign in" %}</title>
+    <title>{% translate "Paperless-ngx sign in" %}</title>
```

Comment:
I think this change will mess up Crowdin

Reply:
Not really, it'll change this item to be a different item that needs translation but thats fine, it should be that way. And then I think it'll take the translation from elsewhere since its the same string

Reply:
Ah, youre right tho will need to regenerate the strings file but thats easy, 1 sec. Let me make sure its not crazy

Reply:
Okay I'll defer to you on that

Reply:
Oh wait duh thats Django template, same idea but let me look at that

Reply:
OK famous last words here but I think this should be fine, updated `django.po` and thats the "source" for all others, should flow via Crowdin when this is merged into `dev`

*should* 😅

Reply:
Looks like these changes got lost maybe in the Schnuffle stuff. I’ll sort it out in a bit

Reply:
> Famous last words here

Thanks :smile: